test/postgres/providers.tf

```diff
@@ -0,0 +1,9 @@
+variable "aws_region" {
+  default = "us-west-2"
+}
+
```

Comment:
Is there a reason to make the `test` directory specifically over just putting provider info at a file at the root of the project instead?

Reply:
I like the test directory for the module testing since it makes it easier to do individual tests as needed.  Though the provider info could certainly be at project root with a symlink to each test directory.